src/helpers/ring/mod.rs

```diff
@@ -0,0 +1,43 @@
+//! This module contains implementations and traits that enable MPC helpers to communicate with
```

Comment:
lets rebase this PR on top of `main` to exclude things that don't belong here

Reply:
actually, this is latest with main. I moved some stuff around.

`ring.rs` used to have all the code in one file, including the mock module. i brought that into a folder and separated things out, so i can add an `http` implementation

Reply:
gotcha. It is totally fine to do but generally I'd recommend to defer moving things around so we can just focus on reviewing new code. Once we finalize and agree it, we can submit another PR that better organizes the content - usually such changes do not require careful review compared to functional PRs